src/table/components/head/ColumnAdjuster.tsx

```diff
@@ -56,13 +56,51 @@ const ColumnAdjuster = ({ column, isLastColumn }: AdjusterProps) => {
     document.addEventListener('mouseup', mouseUpHandler);
   };
 
+  const focusHeadMenuButton = (event: React.KeyboardEvent | React.FocusEvent<HTMLDivElement>) => {
```

Comment:
this would make sense if it lives in a11y utils file

Reply:
Sure, I will do that after https://github.com/qlik-oss/sn-table/pull/885 merged